config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  get "/questions", to: "questions#get_questions"
```

Comment:
We'll want to stick to the rails standards for the named methods in controllers unless we have special cases. In this case it should be `questions#index`

Reply:
```suggestion
  get "/questions", to: "questions#index"
```

The controller will also have to be updated.